specs/devexperience/DevExperience.md

```diff
@@ -0,0 +1,181 @@
+# Reunion Developer Experience
+
+## 1.0 - Overview
+This spec will detail the design of how the Reunion developer experience will work. Through analysis of the current landscape of Windows development, there are a few important items that were found:
+1.	The Windows Application Packaging (WAP) project increases build time by 40%
+2.	Requiring the WAP creates a disjointed developer experience, where developers trying to create Win32 apps have a different workflow than UWP developers.
+3.	Therefore, to provide our customers with the best inner-loop, and to unify the experience for Win32 and UWP developers, we must not require the WAP for our core user scenarios.
+
+If we are to remove the WAP project, we then must ask ourselves the following questions:
+1.	What kicks in the MSIX Package tooling?
+2.	Can developers opt-out of the pri generation?
+3.	How does building a Sparse Package look?
+    - A Sparse Package is a Win32 app that has identity, but isn’t using MSIX.
```

Comment:
> Not quite. A 'sparse package' is a package that contains 'footprint' files but no 'content'. It's typically used in conjunction with an ExternalLocation telling Windows where 'content' files can be found.

This may have been worded poorly because it is using an empty msix file. But I think the below definition is simpler to understand.